java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:101)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:81)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:66)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:101)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:81)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:66)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:101)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:81)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:66)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:101)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:81)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:66)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:97)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:77)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:62)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:97)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:77)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:62)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:112)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:988)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1056)
	at dao.StatisticsDao.getTotalIncome(StatisticsDao.java:83)
	at controllers.StatisticsController.renderData(StatisticsController.java:77)
	at controllers.StatisticsController.lambda$0(StatisticsController.java:56)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:97)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:77)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:62)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:988)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1056)
	at dao.StatisticsDao.getTotalIncome(StatisticsDao.java:83)
	at controllers.StatisticsController.renderData(StatisticsController.java:77)
	at controllers.StatisticsController.lambda$0(StatisticsController.java:56)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.sql.SQLSyntaxErrorException: Table 'java_user_database.order' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:112)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.StatementImpl.executeQuery(StatementImpl.java:1312)
	at dao.StatisticsDao.getSalaryEmployee(StatisticsDao.java:271)
	at controllers.EmployeeStatisticsController.renderEmployee(EmployeeStatisticsController.java:97)
	at controllers.EmployeeStatisticsController.renderData(EmployeeStatisticsController.java:77)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:62)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:112)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.RevenueStatisticsView.showError(RevenueStatisticsView.java:57)
	at controllers.StatisticsController.lambda$0(StatisticsController.java:52)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:112)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:112)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:115)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
java.lang.Exception: Ngày bắt đầu không thể sau ngày kết thúc
	at views.EmployeeStatisticsView.showError(EmployeeStatisticsView.java:122)
	at controllers.EmployeeStatisticsController.lambda$0(EmployeeStatisticsController.java:58)
	at utils.Debouncer$1.run(Debouncer.java:24)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1575)
